docs/frontend_readme.md

```diff
@@ -35,7 +43,7 @@ The betterangels_frontend is built in [React Native](https://reactnative.dev/).
    [ -s "/usr/local/opt/nvm/etc/bash_completion.d/nvm" ] && \. "/usr/local/opt/nvm/etc/bash_completion.d/nvm"
    ```
 
-   If these instructions don't appear, run the following:
+   If these instructions don't appear, run the following: (what do you mean @paul?)
```

Comment:
@vecchp what does this line mean?

Reply:
I can't say I wrote this line (though git history may say so since we just rewrote it completely). I believe it originally came from @chris920.

From what I can tell by reading it, the intention is as follows:
* When installing nvm for the first time with brew, certain lines are emitted into the terminal. If these lines don't appear, the user should type brew info nvm to retrieve them.

Reply:
aw ok thanks :) hi @chris920!!!